Global/EnemyAction.gd

```diff
@@ -0,0 +1,29 @@
+extends RefCounted
+
+class_name EnemyAction
+
+var caster: Entity
+var action: CardBase
+var target: Array[Entity]
```

Comment:
this should not be named target (singular) when it's an array potentially containing multiple targets
`list_targets` might be better